tests/phpunit/tests/theme/wpGetGlobalStylesheet.php

```diff
@@ -270,4 +270,16 @@ public function test_should_enqueue_stored_styles() {
 			'Registered styles with handle of "wp-style-engine-my-styles" do not match expected value from the Style Engine store.'
 		);
 	}
+	/**
+	 * Tests that switching themes recalculates the stylesheet.
+	 *
+	 * @ticket 56970
+	 */
+	public function test_switching_themes_should_recalculate_stylesheet() {
+		$stylesheet_for_default_theme = wp_get_global_stylesheet();
+		switch_theme( 'block-theme' );
```

Comment:
Shouldn't we switch back to the default at the end of the test?

Reply:
Nicely spotted @mcsf!

Reply:
Nice. Done at https://github.com/WordPress/wordpress-develop/pull/3712/commits/09ccca5646d446ad3da740675855bd24c37c53d1